### PR TITLE
11.0 beta history seed

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

**Comment by jahorton** [comment]
Running checks on each section's history.md with resources/validateHistory.sh:

macOS:  It wasn't caught before, but we have an error in 10.0.29's alpha section - particularly, that it reports the ".29" portion.

I also noted a couple issues to be resolved with the script, but that's probably on me:

Android:  reports errors with 2.0 - 2.4 entries, but those should just be warnings 'cause that's the grandfathered section of history.md.

Linux:  doesn't yet process its history.md because it doesn't look for it; simply adding "linux" to the array on line 32 of resources/shellHelperFunctions.sh will fix that, though.  (And then its history.md validates fine.)

**Comment by mcdurdin** [comment]
@jahorton Good catches. I think we need to have a trigger in TC that runs only if history.md is updated and generates a report of these items. Would you like to push changes that you noted to the script to this PR (if you like, you could patch the macOS history.md issue too)?

I will setup the TC build configuration and the trigger and test it on this PR.

**Comment by jahorton** [comment]
Alright, looks like those history-validation issues are now fixed, and that TC build configuration is now in place.  No reason to block on those points now.

**Comment by mcdurdin** [comment]
Version numbers should be 3-number not 4-number for some platforms...

File: web/history.md [hunk 1]
```diff
@@ -1,12 +1,26 @@
 # KeymanWeb Version History
 
-## 11.0 alpha
-* Add `setNumericLayer()` for embedded platforms to change OSK to numeric layer.
-* Fixes issue where file extensions are upper-case, e.g. ".TTF"
-* Fixes keyboard layout issues after mobile device rotations. (#248) (#970)
-* Fixes issue with oversized key text on some keyboards. (#382)
-* Fixes issue with diacritics not displaying on some keyboards.  (#1070)
-* Adds support for Promises to init() and setActiveKeyboard(). (#100)
+## 2019-01-02 11.0.1300.0 beta
```
Comment (mcdurdin) [inline web/history.md]: Should be `11.0.200`

File: ios/history.md [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
-* Replaced deprecated calls to UIAlertView and cleaned up extraneous blank buttons (#1002)
-* Bookmark add button is enabled only when title/url fields have text (#1073)
-* Removed deprecated code to to support keyman:// scheme for ad-hoc distribution (#1160)
-* Updated the default keyboard to SIL EuroLatin (#1288)
-* Fixed bug behind some crashes of system keyboard (#1166)
-* Added support for keypress error feedback (#257)
-* Fixed ongoing issues with keyboard rotation and sizing, including the iPhone X notch. (#444) (#1045)
-* Fixed keyboard display/overlap of "Getting Started" info panel, added keyboard hide/display API functions. (#1084)
-* Fixes issues with keyboard keycap scaling and diacritic display. (#1070)
+## 2019-01-02 11.0.200.0 beta
```
Comment (mcdurdin) [inline ios/history.md]: Should be `11.0.300`

**Pull request merged**